frontend/src/components/notifications/styles.module.scss

```diff
@@ -7,7 +7,8 @@
 
 .containerUserNotFound {
   width: 100%;
-  height: 100%;
+  margin-top: 350px;
```

Comment:
Is it for locating on the middle? It will be better using either `display: flex; justify-content: center; align-items: center` or `position: absolute` with offset in %. Because different phones have different screen sizes and therefore `350px` will look different.